.bazelrc

```diff
@@ -10,6 +10,14 @@ build:android_arm64 --config=android
 build:android_arm64 --cpu=arm64-v8a
 build:android_arm64 --fat_apk_cpu=arm64-v8a
 
+# This is required for the default Android emulators, right? 
```

Comment:
Actually, sorry for the last minute request, can you remove this comment and simply group these new rule with the existing arm rules?

Reply:
Sure, I can do that.